### PR TITLE
Using ENV variable instead of hardcodded 'kubectl' in Makefiles.

### PR DESCRIPTION
**Pull request by ybettan**
We want to be able to support 'oc' in addition to 'kubectl'.

Also 'oc' was set as default.
We can change it to 'kubectl' once we go upstream together with
CONTAINER_COMMAND which default to 'podman' and should be changed to
'docker' when we move upstream.

Signed-off-by: Yoni Bettan <ybettan@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,27 +106,27 @@ generate-mocks:
 ##@ Deployment
 
 install: manifests kustomize  ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+	$(KUSTOMIZE) build config/crd | $(CLUSTER_CLINET) apply -f -
```
Comment (yevgeny-shnaidman) [inline Makefile]: should not it be CLUSTER_CLIENT?

Comment (ybettan) [reply Makefile]: Good catch, fixed it.

**Pull request merged**